src/loading-bar/styles.scss

```diff
@@ -0,0 +1,59 @@
+@use "../../node_modules/@cloudscape-design/design-tokens/index.scss" as awsui;
```

Comment:
```suggestion
@use "~@cloudscape-design/design-tokens/index.scss" as awsui;```

Reply:
we don't currently have the right build tool config for this (see also e.g. https://github.com/cloudscape-design/board-components/blob/main/src/board/styles.scss#L1), will create a follow up to try to tidy this up across all sub-repositories